lib/ansible/cli/pull.py

```diff
@@ -116,8 +112,7 @@ def parse(self):
         self.parser.add_option('--accept-host-key', default=False, dest='accept_host_key', action='store_true',
                                help='adds the hostkey for the repo url if not already added')
         self.parser.add_option('-m', '--module-name', dest='module_name', default=self.DEFAULT_REPO_TYPE,
-                               help='Repository module name, which ansible will use to check out the repo. Choices are %s. Default is %s.'
-                                    % (self.REPO_CHOICES, self.DEFAULT_REPO_TYPE))
```

Comment:
Why is the error message changed here ?

Reply:
My mistake. I think I unknowingly changed that. I will update PR to keep the previous message. @mscherer Thank you.